src/test/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorWorkerUnitTest.java

```diff
@@ -1,138 +0,0 @@
-package org.broadinstitute.hellbender.dev.tools.walkers.bqsr;
-
-import htsjdk.samtools.TextCigarCodec;
-import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
-import org.broadinstitute.hellbender.utils.read.GATKRead;
-import org.broadinstitute.hellbender.utils.recalibration.EventType;
-import org.broadinstitute.hellbender.utils.test.BaseTest;
-import org.testng.Assert;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-public final class BaseRecalibratorWorkerUnitTest extends BaseTest {
```

Comment:
Does deleting `BaseRecalibratorWorkerUnitTest` and `CalibrationTablesBuilderIntegrationTest` reduce our test coverage at all (ie., are we no longer testing features that used to be tested?)